.github/workflows/publish.yml

```diff
@@ -0,0 +1,66 @@
+---
+name: Publish
+on:
+  push:
+    tags:
+      - "v*"
+env:
+  ALIAS: aquasecurity
+  REP: harbor-scanner-trivy
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Check Out Repo
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildxarch-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildxarch-
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Login to ECR
+        uses: docker/login-action@v1
+        with:
+          registry: public.ecr.aws
+          username: ${{ secrets.ECR_ACCESS_KEY_ID }}
+          password: ${{ secrets.ECR_SECRET_ACCESS_KEY }}
```

Comment:
Hey @krol3 

I think that we could just take these two login steps for DockerHub and ECR and add them to the existing release workflow, which is triggered whenever we create an annotated Git tag (e.g. `git tag -a v1.4.0 -m 'Release v1.4.0'`):
https://github.com/aquasecurity/harbor-scanner-trivy/blob/main/.github/workflows/release.yml#L29

We're also using GoRelease, which builds Docker image and adds metadata labels. It also allows you to add ECR image template in https://github.com/aquasecurity/harbor-scanner-trivy/blob/main/.goreleaser.yml#L29

Reply:
Great idea!! I will update the PR

Reply:
Follow the test [here](https://github.com/krolval/harbor-scanner-trivy/actions/runs/440965817)